### PR TITLE
Change "Recovery certificate(s)" to "Certificate(s) of recovery"

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR changes all occurrences of "Recovery certificate(s)" with "Certificate(s) of recovery", as this is the term used in the app.

## What was changed?

The term "Recovery certificate(s)" was replaced with "Certificate(s) of recovery" in some FAQ & glossary entries.

## Screenshots

I hope it is acceptable that I didn't make a screenshot of every change (-:

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -2869,9 +2873,9 @@
                     "description": "Since version 2.2 of the Corona-Warn-App you are able to create a rapid test profile with your data such as first and last name, date of birth, address, phone number and e-mail address. Using a rapid test profile, you ensure that registration at a testing center is faster. The profile contains a QR code, which you can show at the testing center. However, you should still have your ID card ready."
                 },
                 {
-                    "term": "Recovery Certificate",
-                    "anchor": "recovery_certificate",
-                    "description": "The certificate of recovery is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                    "term": "Recovery certificate",
+                    "anchor": "recovery_certificate_alt",
+                    "description": "Please see the gelossary entry <a href='#recovery_certificate'>Certificate of Recovery</a>."
```
Comment (MikeMcC399) [inline src/data/faq.json:2878]: "gelossary" is misspelled

**Comment by MikeMcC399** [comment]
@Ein-Tim 
The previous PR https://github.com/corona-warn-app/cwa-website/pull/2263 was https://github.com/corona-warn-app/cwa-website/labels/on%20hold, so I'd like to see feedback from the Open Source Team about their intentions before spending time on reviewing in detail.

I have just commented on one typo so far.

**Comment by Ein-Tim** [comment]
@MikeMcC399 For sure!

File: src/data/faq.json [hunk 2]
```diff
@@ -610,7 +609,7 @@
...
                             "If the rules established in accordance with the respective current recommendations of the Standing Committee on Vaccination at the Robert Koch Institute (STIKO) apply to your certificates stored in the app, the app will notify you of the corresponding vaccination recommendation. Please then discuss with your doctor(s) whether you should receive another vaccine dose of a COVID-19 vaccine.",
-                            "If the assumptions derived from your saved certificates do not apply because you have already received further COVID-19 vaccinations or have undergone a proven SARS-CoV-2 infection, you can ignore this notification. In this case, we recommend scanning the relevant vaccination or recovery certificates into the app.",
+                            "If the assumptions derived from your saved certificates do not apply because you have already received further COVID-19 vaccinations or have undergone a proven SARS-CoV-2 infection, you can ignore this notification. In this case, we recommend scanning the relevant vaccination or certificates of recovery into the app.",
```
Comment (MikeMcC399) [inline src/data/faq.json:612]: This change does not work grammatically. I suggest reverting to the original text.

File: src/data/faq.json [hunk 3]
```diff
@@ -2869,9 +2873,9 @@
                     "description": "Since version 2.2 of the Corona-Warn-App you are able to create a rapid test profile with your data such as first and last name, date of birth, address, phone number and e-mail address. Using a rapid test profile, you ensure that registration at a testing center is faster. The profile contains a QR code, which you can show at the testing center. However, you should still have your ID card ready."
                 },
                 {
-                    "term": "Recovery Certificate",
-                    "anchor": "recovery_certificate",
-                    "description": "The certificate of recovery is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                    "term": "Recovery certificate",
```
Comment (MikeMcC399) [inline src/data/faq.json:2876]: Revert to the original capitalization "Recovery Certificate" using title case inline with other glossary entries.

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq.json:2878]: Also this is a dead link. It should be #glossary_recovery_certificate

File: src/data/faq.json [hunk 4]
```diff
@@ -2935,7 +2939,7 @@
                 {
                     "term": "Technical Validity",
                     "anchor": "technical_validity",
-                    "description": "The certificate as a document has an issue date and a technical expiration date, just like your passport or ID card, for example. For all certificates, the expiration date is set by the issuer of the certificate. Currently in Germany, the EU digital vaccination, test and recovery certificates are usually issued with a technical validity of one year from the date of issue."
+                    "description": "The certificate as a document has an issue date and a technical expiration date, just like your passport or ID card, for example. For all certificates, the expiration date is set by the issuer of the certificate. Currently in Germany, the EU digital vaccination, test and certificates of recovery are usually issued with a technical validity of one year from the date of issue."
```
[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).

File: src/data/faq.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).

File: src/data/faq.json [hunk 5]
```diff
@@ -403,7 +402,7 @@
...
-                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?</a></li><li><a href='#admission_policy_2gplus_howto' target='_blank'>How do I use the combined 2G+ display?</a></li></ul>"
+                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certififcate or certificate of recovery and a negative test?</a></li><li><a href='#admission_policy_2gplus_howto' target='_blank'>How do I use the combined 2G+ display?</a></li></ul>"
```
[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Thank you for the review Mike, I addressed your findings in [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`507e44b`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/507e44b7918968071818c73a68239dcbecd904dd).

File: src/data/faq.json [hunk 6]
```diff
@@ -375,20 +374,20 @@
...
+                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certififcate or a certificate of recovery and a negative test?</a></li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:386]: "certififcate" typo

Comment (Ein-Tim) [reply src/data/faq.json]: Thank you! Fixed in [`63367c7`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/63367c733bb083f892e8cba581a5122b81f4ad8e).

**Review by MikeMcC399 — APPROVED** [review]
@Ein-Tim 
LGTM!

I spell-checked the resulting public/en/faq/index.html file and found some other typos, but no more issues in this PR.

**Comment by Ein-Tim** [comment]
Thank you @MikeMcC399!

@dsarkar It was quite an effort to resolve all the merge conflicts, so I would be very happy if you could merge this soon! 

Thanks!

**Review by dsarkar — APPROVED** [review]

**Comment by Ein-Tim** [comment]
@dsarkar There are now many unwanted commits in here, if you don't want this I can provide a clean PR. However, if you use squash and merge, there should not be a problem.

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks!

**Pull request merged**